app/src/main/java/com/example/movies/network/Movie.kt

```diff
@@ -1,13 +1,26 @@
 package com.example.movies.network
 
 import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class Movies(
+    @field:Json(name = "results") val movies: List<Movie>
+)
 
 data class Movie(
-        @Json(name = "poster_path") val imgSrcUrl: String,
-        val overview: String,
-        val release_date: String,
-        val id: Long,
-        val original_language: String,
-        val title: String,
-        val popularity: Float
+    @Json(name = "poster_path") val posterPath: String,
+    @Json(name = "adult") val adult: Boolean,
+    @Json(name = "overview")val overview: String,
+    @Json(name = "release_date")val releaseDate: String,
```

Comment:
Without the nullable operator (?) on the releaseDate the app doesn't display any movie, It seem to be that this field it could be empty, if you can added it would be great